packages/website/src/components/row.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import css from 'classnames';
+import styled from 'styled-components';
+
+const RowContainer = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+const RowComponent = styled.div`
+  margin-left: var(--space-half-vw);
+`;
+
+const Row = ({ className, children }) => {
+  return (
+    <RowContainer className={css(className)}>
```

Comment:
We should not use `className` anywhere, instead we want to work only with the `styled` API

Reply:
okay I'll fix it